MaxText/layers/models.py

```diff
@@ -145,6 +146,16 @@ class Decoder(nn.Module):
   shared_embedding: nn.Module
   mesh: Mesh
 
+  def get_decoder_layer(self):
+    if self.config.model_name == "default":
+      return DecoderLayer
+    elif self.config.model_name[0:5] == "llama":
```

Comment:
Sorry to be the fun police but llama2 please! (or if we only support llama2-7b let's say it.